app/assets/stylesheets/wcag.scss

```
@@ -280,7 +286,46 @@ html {
     color: var(--primary-medium);
   }
 
-  // Admin
+  .alert.alert-success a {
+    color: dark-light-choose($tertiary, $quaternary-high);
+  }
+
+  // User Preference
```

```suggestion
  // User Preferences
```

Can you post before/after screenshots for this as well, please?